docs/crio.conf.5.md

```diff
@@ -284,7 +284,7 @@ CRI-O reads its configured registries defaults from the system wide containers-r
   List of registries to skip TLS verification for pulling images.
 
 **registries**=["docker.io"]
-  List of registries to be used when pulling an unqualified image (e.g., "alpine:latest"). By default, registries is set to "docker.io" for compatibility reasons. Depending on your workload and usecase you may add more registries (e.g., "quay.io", "registry.fedoraproject.org", "registry.opensuse.org", etc.).
+  List of registries to be used when pulling an unqualified image. Note that this option has been deprecated and has no effect. Please refer to `containers-registries.conf(5)` for configuring unqualified-search registries.
```

Comment:
The following is a soft suggestion/thought for the below.  Use if you think any of this is better.

IDK, but maybe this will draw the eye better?  Or some variant?   Perhaps "Deprecated as of v1.20."  Regardless, I don't think you need the "and has no effect." bit.

```suggestion
  List of registries to be used when pulling an unqualified image. DEPRECATED.  Please refer to `containers-registries.conf(5)` for configuring unqualified-search registries.
```

Reply:
I tend to understand “deprecated” as “discouraged but continues to work”; if it is not just deprecated but basically removed, I think that should be documented explicitly. (Or, maybe, just remove the option documentation entirely.)

OTOH I don’t see much value in documenting what the option did _in the past_; that can distract readers from reading on to notice that it doesn’t do that any more.

Reply:
In this PR I think we should drop the entry entirely, and mark the deprecation notice in 1.20

Reply:
@haircommander dropping would be fine with me.  I was just trying to nitpick something based on the prior comments.

Reply:
I concur with @mtrmac's understanding of deprecated.  The important part is that "it takes no effect".  I see wisdom in helping users resolve a problem.  They may not (always) look at the logs.

Reply:
I'll drop it :)

Reply:
sorry my brain is all over the place. we can leave this in for a PR, and I'd suggest the wording: 
```
List of registries to be used when pulling an unqualified image. Note that support for this option has been dropped and setting it has no effect. Please refer to `containers-registries.conf(5)` for configuring unqualified-search registries.
```

Reply:
SGTM :)